daemon/algod/api/algod.oas2.json

```diff
@@ -1385,6 +1385,17 @@
               "type": "string",
               "format": "binary"
             }
+          }, 
+          {
+            "name": "sourcemap",
+            "description": "When set to `map`, returns the source map of the program as a JSON. Defaults to `nomap`.",
+            "in": "query",
+            "required": false,
+            "type": "string",
```

Comment:
I think this would be better off as a boolean, unless you think it could make sense to accept more than 2 values here.

You probably modeled this off the `exclude` parameter from `/v2/accounts/{address}`, which is a special case. There's also an `exclude` parameter in indexer that accepts more than 2 values, and for consistency I think algod's was named the same thing.

Reply:
Makes sense, thanks for the explanation behind it as well!